src/nav/telemetry-data-platform.yml

```diff
@@ -61,6 +61,8 @@ pages:
         path: /docs/telemetry-data-platform/manage-data/manage-data-retention
       - title: Drop data
         path: /docs/telemetry-data-platform/manage-data/drop-data-using-nerdgraph
```

Comment:
I do feel that the NrIntegrationLimits errors doc should go after the Limits docs in the nav. They talk about concepts of limits. This NrIntegrationLimits errors doc is more like a reference in my opinion. It talks about the various queries you can run. You would need the System limits doc for context first.

Reply:
@mmfred making sure we're on same page (maybe we can talk about it at our meeting later): the NrIntegrationError event is about more than limits; it's also about malforming of data, error codes related to ingest, config issues. It's primarily about data limits and other ingest problems right now, but it's def going to have more additions in functionality later, is the idea. So I'm making sure we're on same page because I don't think of NrIntegrationError as subset of limits but the other way around. Or rather, they're maybe separate topics, with NrIntegrationError just being the way that limits (and other things) are exposed/captured.

Reply:
having said that, I can see it going last in that category because it is more of a reference and the limits docs will link to it. I just wanted to make sure you saw all that context and still want it last.

Reply:
fyi, I did move this to end of category for my latest edit.